lib/lotus/generators/application/container/Gemfile.tt

```diff
@@ -16,6 +16,16 @@ gem 'lotusrb',      '<%= Lotus::VERSION %>'
 gem 'lotus-model', '<%= config[:lotus_model_version] %>'
 <%- end -%>
 
+<%- if config[:adapter] == 'mysql' %>
```

Comment:
erb has trouble with case syntax. You need to put the `case` and first `when` in the same block. Like this

```ruby
<%- case config[:adapter]
    when 'mysql' %>
gem 'mysql'
<%- when 'mysql2' %>
gem 'mysql2'
<%- when 'postgresql' %>
gem 'pg'
<%- when 'sqlite3' %>
gem 'sqlite3'
<%- end -%>
```

Looks kind of broken to me :)